app/controllers/admin/content/system-images/subtopics.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+const { Controller, computed } = Ember;
+export default Controller.extend({
+  subTopics: computed(function() {
```

Comment:
Like I said ... this computed property is not needed anymore ... Directly use the `model`